### PR TITLE
docs: Clarification and edits in Data availability

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Clarification and edits in Data availability

### PR Preview URL

[Data availability](https://starknet-io.github.io/starknet-docs/pr-949/documentation/architecture_and_concepts/Network_Architecture/on-chain-data/)

### Check List

- [x] Changes have been done against dev branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/949)

### CODE REVIEW

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc [hunk 1]
```diff
@@ -4,40 +4,45 @@
...
+== Data availability: v0.11.0 and forward
 
 
 [id="v0.11.0_format"]
 === v0.11.0 format
 
 The state diffs contain information on every contract whose storage was updated and additional information on contract deployments.
 
-For each affected contract, we have:
+For each affected contract the following information is sent to the Ethereum core contract in a transaction as calldata:
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc:27]:
```suggestion
For each affected contract, the following information is sent as calldata on L1:
```

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc [hunk 2]
```diff
@@ -4,40 +4,45 @@
 [id="introduction"]
 == Introduction
 
-In the current stage of the Alpha, Starknet operates in a ZK-Rollup mode. This means that upon the acceptance of a state update onchain, the state diff between the previous and new state is sent as calldata to Ethereum.
+In the current stage of the Alpha, Starknet operates in Validity Rollup mode, also referred to as ZK Rollup mode. This means that upon the acceptance of a state update onchain, the state diff between the previous and new state is sent as calldata to Ethereum.
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc:7]:
```suggestion
In the current stage of the Alpha, Starknet operates in Validity Rollup mode, also referred to as ZK-Rollup mode. This means that upon the acceptance of a state update onchain, the state diff between the previous and new state is sent as calldata to Ethereum.
```

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc [hunk 3]
```diff
@@ -78,23 +82,15 @@ according to the xref:#v0.11.0_format[above format].
...
-xref:starknet_versions:upcoming_versions.adoc#what_to_expect[compiled class hash] of the declared
-class
+* The first element, `1`, is the number of contracts whose state was updated.
+* The second element, `2019172390095051323869047481075102003731246132997057518965927979101413600827`, is the address of the first, and only, contract whose state changed.
+* The third element, `18446744073709551617`, which is 2^64^+1, encodes the following:
+** The class information flag is `0`, that is, the contract was not deployed or replaced just now, so you shouldn't treat the next word as the class hash.
+** The new nonce is `1`.
+** One storage cell was updated.
+* The next two elements, `100` and `200`, encode the storage update, where the value of key `100` is set to `200`.
+* he new declare section: `1` includes a single xref:Network_Architecture/transactions.adoc#declare_v2[declare v2] transaction in this state update, and the next two elements
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/on-chain-data.adoc:92]:
```suggestion
* the new declare section: `1` includes a single xref:Network_Architecture/transactions.adoc#declare_v2[declare v2] transaction in this state update, and the next two elements
```

**Review by ArniStarkware — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r4, 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/949#-:-NkK02Sx44fUxQdshUzw:bnfp4nl)* status: all files reviewed (commit messages unreviewed), 3 unresolved discussions

**Review by ArniStarkware — APPROVED** [review]
> Reviewed all commit messages.
> Dismissed @stoobie from a discussion.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/949)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @stoobie)

**Pull request merged**